regridding/slurm.py

```diff
@@ -184,29 +206,36 @@ def parse_args():
     # build and write sbatch files
     sbatch_fps = []
     sbatch_dir = slurm_dir.joinpath("regrid")
-    # remove any existing slurm jobs in this directory
-    _ = [fp.unlink() for fp in sbatch_dir.glob("*.slurm")]
     sbatch_dir.mkdir(exist_ok=True)
-    for fp in regrid_batch_dir.glob("*.txt"):
-        sbatch_str = fp.name.split("batch_")[1].split(".txt")[0]
-        sbatch_fp = sbatch_dir.joinpath(f"regrid_{sbatch_str}.slurm")
-        # filepath for slurm stdout
-        sbatch_out_fp = sbatch_dir.joinpath(sbatch_fp.name.replace(".slurm", "_%j.out"))
-
-        sbatch_head = make_sbatch_head(slurm_email, conda_init_script)
-        sbatch_regrid_kwargs = {
-            "sbatch_fp": sbatch_fp,
-            "sbatch_out_fp": sbatch_out_fp,
-            "regrid_script": regrid_script,
-            "regrid_batch_dir": regrid_batch_dir,
-            "regrid_dir": regrid_dir,
-            "regrid_batch_fp": fp,
-            "dst_fp": target_grid_fp,
-            "no_clobber": no_clobber,
-            "sbatch_head": sbatch_head,
-        }
-        write_sbatch_regrid(**sbatch_regrid_kwargs)
-        sbatch_fps.append(sbatch_fp)
+
+    # remove any existing sbatch files in this directory.
+    #  Easier to keep track of things when the only jobs are those submitted from this directory.
+    _ = [fp.unlink() for fp in sbatch_dir.glob("*.slurm")]
+
+    for var in vars.split():
+        for freq in freqs.split():
+            for fp in regrid_batch_dir.glob(f"*{var}*{freq}*.txt"):
```

Comment:
```suggestion
            for fp in regrid_batch_dir.glob(f"*{freq}*{var}*.txt"):
```

I needed to make the change above (swap the order of `{var}` and `{freq`) to get this to run properly for me. Otherwise it wasn't finding any files to regrid.

Reply:
Ah, yes! Fixed in 07de0374